src/main/java/cms/gov/madie/measure/config/mongock/UpdateMeasureOrganizationRecordNameChangeUnit.java

```diff
@@ -0,0 +1,33 @@
+package cms.gov.madie.measure.config.mongock;
+
+import cms.gov.madie.measure.repositories.OrganizationRepository;
+import io.mongock.api.annotations.ChangeUnit;
+import io.mongock.api.annotations.Execution;
+import io.mongock.api.annotations.RollbackExecution;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@ChangeUnit(id = "measure_orgs_update_record_name", order = "1", author = "madie_dev")
+public class UpdateMeasureOrganizationRecordNameChangeUnit {
+
+    // this method with updated filter conditions and change unit ID can be used for future updates
+  @Execution
+  public void updateMeasureOrganizationRecordName(OrganizationRepository organizationRepository) {
+    log.info("Entering the organizations for updating the The Joint Commission name");
+
+    organizationRepository.findAll().stream()
```

Comment:
How about just updating one conditionally instead of fetching all? 
```
Query query = new Query(Criteria.where("name").is("The Joint Commission"));
 Update update = new Update().set("name", "Joint Commission");
var result = mongoTemplate.updateFirst(query, update, "organization");
```